bindings.c

```diff
@@ -1909,18 +1919,25 @@ int cg_open(const char *path, struct fuse_file_info *fi)
 
 int cg_access(const char *path, int mode)
 {
+	int ret;
 	const char *cgroup;
-	char *last = NULL, *path1, *path2, * cgdir = NULL, *controller;
+	char *path1, *path2, *controller;
+	char *last = NULL, *cgdir = NULL;
 	struct cgfs_files *k = NULL;
 	struct fuse_context *fc = fuse_get_context();
-	int ret;
+
+	if (strcmp(path, "/cgroup") == 0) {
+		if ((mode & W_OK) == 0)
```

Comment:
Yeah, I think I'll remove the `mode & W_OK` check for the reason outlined above.